### PR TITLE
Pop up card

### PR DESCRIPTION
**Pull request by turcottep**

### CODE REVIEW

File: cryptus-app/components/profile/profile.tsx [hunk 1]
```diff
@@ -35,12 +39,57 @@ export default function Profile(props: {
     console.log("is_my_profile", is_my_profile);
   }, [loading]);
 
+  const close_all = () => {
+    console.log("close_all");
+
+    set_show_collection(-1);
```
Comment (alexandrelafleur) [inline cryptus-app/components/profile/profile.tsx]: use 2 states instead of 1

File: cryptus-app/components/utils/card/card.module.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
+@use "/styles/variables" as v;
+
+.container {
+  flex: 0 0 none;
+  z-index: 100;
+  width: 512px;
```
Comment (alexandrelafleur) [inline cryptus-app/components/utils/card/card.module.scss]: make bigger width, and use @media

File: cryptus-app/components/utils/card/card.module.scss [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  align-items: center;
+  min-width: 100vw;
+  min-height: 100vh;
+  z-index: 10;
+  backdrop-filter: blur(4px);
+  background-color: rgba(0, 0, 0, 0.2);
+  cursor: pointer;
+}
+
+.x_button {
```
Comment (alexandrelafleur) [inline cryptus-app/components/utils/card/card.module.scss:43]: remove when desktop

File: cryptus-app/components/utils/card/card.tsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        viewBox="0 0 24 24"
+        stroke="currentColor"
+        strokeWidth={3}
+      >
+        <path
+          strokeLinecap="round"
+          strokeLinejoin="round"
+          d="M6 18L18 6M6 6l12 12"
+        />
+      </svg>
```
Comment (alexandrelafleur) [inline cryptus-app/components/utils/card/card.tsx:50]: svg in /public

File: cryptus-app/components/wallet_viewer/collection_details/collection_details.module.scss [hunk 5]
```diff
@@ -1,8 +1,31 @@
...
 .container {
-  display: flex;
-  flex-direction: column;
+  flex: 0 0 none;
+  z-index: 100;
+  width: 512px;
+  padding: 16px;
+  border-radius: 16px;
+  position: relative;
+  margin-top: v.$header-h;
```
Comment (alexandrelafleur) [inline cryptus-app/components/wallet_viewer/collection_details/collection_details.module.scss]: why

File: cryptus-app/components/wallet_viewer/collection_details/collection_details.module.scss [hunk 6]
```diff
@@ -1,8 +1,31 @@
...
 
 .container {
-  display: flex;
-  flex-direction: column;
+  flex: 0 0 none;
+  z-index: 100;
+  width: 512px;
+  padding: 16px;
+  border-radius: 16px;
+  position: relative;
```
Comment (alexandrelafleur) [inline cryptus-app/components/wallet_viewer/collection_details/collection_details.module.scss]: why is the card logic here 2 times

File: cryptus-app/components/wallet_viewer/collection_details/collection_details.module.scss [hunk 7]
```diff
@@ -1,8 +1,31 @@
...
-  width: 100%;
+  background-color: v.$bg-color;
+  cursor: default;
+}
+
+.backdrop {
+  position: fixed;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (alexandrelafleur) [inline cryptus-app/components/wallet_viewer/collection_details/collection_details.module.scss]: this is not card

File: cryptus-app/components/wallet_viewer/nft_details/nft_details.module.scss [hunk 8]
```diff
@@ -14,3 +14,16 @@
...
+.backdrop {
+  position: fixed;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-width: 100vw;
+  min-height: 100vh;
+  z-index: 10;
+  backdrop-filter: blur(4px);
+  background-color: rgba(0, 0, 0, 0.2);
```
Comment (alexandrelafleur) [inline cryptus-app/components/wallet_viewer/nft_details/nft_details.module.scss]: `#000, 0.2

File: cryptus-app/pages/[userId]/index.tsx [hunk 9]
```diff
@@ -20,9 +20,9 @@ export default function post(props) {
   }, [mobile]);
 
   return (
-    <AnimatedDiv>
-      <Profile collections={collections} user={user} isMobile={isMobile} />
-    </AnimatedDiv>
+    // <AnimatedDiv>
```
Comment (alexandrelafleur) [inline cryptus-app/pages/[userId]/index.tsx]: find why it doesn't work, and make it work with animatedDiv

File: cryptus-app/__tests__/snapshot/market_overview/market_collection/__snapshots__/market_collection.snapshot.test.tsx.snap [hunk 10]
```diff
@@ -1,157 +1,105 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`<MarketCollection /> should display properly 1`] = `
-<div>
-  <div
```
Comment (alexandrelafleur) [inline cryptus-app/__tests__/snapshot/market_overview/market_collection/__snapshots__/market_collection.snapshot.test.tsx.snap:5]: a

**Review by alexandrelafleur — CHANGES_REQUESTED** [review]

File: cryptus-app/components/utils/card/card.tsx (hunk 4 above)
Comment (turcottep) [reply cryptus-app/components/utils/card/card.tsx:50]: can't change color and strokewidth if in images

File: cryptus-app/components/wallet_viewer/collection_details/collection_details.module.scss (hunk 5 above)
Comment (turcottep) [reply cryptus-app/components/wallet_viewer/collection_details/collection_details.module.scss]: removed whole file

File: cryptus-app/__tests__/snapshot/market_overview/market_collection/__snapshots__/market_collection.snapshot.test.tsx.snap (hunk 10 above)
Comment (turcottep) [reply cryptus-app/__tests__/snapshot/market_overview/market_collection/__snapshots__/market_collection.snapshot.test.tsx.snap:5]: ?

**Pull request merged**